<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataEditor</class>
 <widget class="QDialog" name="MetadataEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Скасувати</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="metadataTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
